src/main/java/com/dreammaster/scripts/ScriptAppliedEnergistics2.java

```diff
@@ -62,7 +63,8 @@ public List<String> getDependencies() {
                 ExtraUtilities.ID,
                 IndustrialCraft2.ID,
                 IronChests.ID,
-                Thaumcraft.ID);
+                Thaumcraft.ID,
+                GTPlusPlus.ID);
```

Comment:
Where is this dep needed?

Reply:
while it is a gt.blockmachines, it comes from gtpp

Reply:
Which one? If thats an world accelerator then they are in core mod

Reply:
damn you are right, thanks for checking. my bad.

Reply:
fixed